packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -287,8 +287,8 @@ export default class CodeMode extends Component<Signature> {
           let currentlyOpenedFile = this.openFile.current as any;
           let cardDoc = JSON.parse(currentlyOpenedFile.content);
           if (isCardDocument(cardDoc)) {
-            let url = currentlyOpenedFile.url.replace(/\.json$/, '');
-            state.value = await this.cardService.loadModel(url);
+            let path: string = currentlyOpenedFile.url.replace(/\.json$/, '');
+            state.value = await this.cardService.loadModel(new URL(path));
```

Comment:
This is a type fix. Because the type for path was `any`, typescript didn't warn when a non-url type was sent in to the `.loadModel` method. This type declaration fixes that issue.